packages/shopify-codemod/test/fixtures/remove-empty-returns/function-declarations.output.js

```diff
@@ -0,0 +1,7 @@
+function whenFinished() {
+  unhighlight(target, startTime);
```

Comment:
My preference would be to have less real world information in the tests.  It's harder to grok when I have to sort out that `whenFinished` and `unhighlight` don't have any significance to the solution.

The project's other tests tend to use `foo`, `bar`, `qux`, and/or convey information about the test.  e.g.,
```js
function returnShouldBeRemoved() {
  foo();
  bar();

  function anotherRemovedReturn() {
    qux();
    return;
  }

  return;
}
```